src/components/base/index.js

```diff
@@ -9,7 +9,8 @@ const BaseToast = (props) => {
   const baseStyle = [
     styles.base,
     styles.borderLeft,
-    { borderLeftColor: props.color || undefined }
+    { borderLeftColor: props.color || undefined },
+    {elevation: props.hideElevation ? 3 : 0},
```

Comment:
Shouldn't this be the other way around?

Reply:
Yep, sorry
I've forgot to rename

I've already fixed